app/models/card.rb

```diff
@@ -1,23 +1,35 @@
 class Card < ActiveRecord::Base
   before_validation :set_review_date, on: :create
+  before_save :set_downcase_text
   validate :original_translated_text_equal
   validates :original_text, :translated_text, :review_date,
             presence: { message: 'Необходимо заполнить поле.' }
 
-  protected
-    def set_review_date
-      self.review_date = Time.now + 3.days
-    end
+  scope :pending, -> { where('review_date <= ?', Time.now).order('RANDOM()') }
 
-    def original_translated_text_equal
-      self.original_text = full_downcase(original_text)
-      self.translated_text = full_downcase(translated_text)
-      if original_text == translated_text
-        errors.add(:original_text, 'Вводимые значения должны отличаться.')
-      end
+  def check_user_translation(user_translation)
+    if translated_text == user_translation.downcase
+      update_attributes(review_date: Time.now + 3.days)
     end
+  end
+
+  protected
+  def set_review_date
+    self.review_date = Time.now + 3.days
+  end
 
-    def full_downcase(str)
-      str.mb_chars.downcase.to_s.squeeze(' ').lstrip
+  def original_translated_text_equal
+    if full_downcase(original_text) == full_downcase(translated_text)
+      errors.add(:original_text, 'Вводимые значения должны отличаться.')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.